build/src/src/pollDappnodeConfig.ts

```diff
@@ -24,14 +24,20 @@ export async function pollDappnodeConfig({
   hostname: string;
   internalIp: string;
 }> {
-  // If ENVs are already available, do not poll
-  const hostnameFromEnv = process.env[GLOBAL_ENVS.HOSTNAME];
-  const internalIpFromEnv = process.env[GLOBAL_ENVS.INTERNAL_IP];
-  if (hostnameFromEnv && internalIpFromEnv)
```

Comment:
We should mantain backwards compatibility here. Also change only the code strictly necessary so there's less risk of breaking stuff since everything works now and won't require that much testing